apiserver/facades/controller/undertaker/undertaker.go

```diff
@@ -67,10 +70,15 @@ func (u *UndertakerAPI) ModelInfo(ctx context.Context) (params.UndertakerModelIn
 		return result, errors.Trace(err)
 	}
 
+	modelInfo, err := u.modelInfoService.GetModelInfo(ctx)
+	if err != nil {
+		return result, errors.Trace(err)
+	}
+
 	result.Result = params.UndertakerModelInfo{
-		UUID:           model.UUID(),
-		GlobalName:     model.Owner().String() + "/" + model.Name(),
-		Name:           model.Name(),
+		UUID:           modelInfo.UUID.String(),
+		GlobalName:     modelInfo.CredentialOwner.String() + "/" + modelInfo.Name,
```

Comment:
this should be `modelInfo.OwnerName.Name() + "/" + modelInfo.Name`

Reply:
in the modelinfo definition, there is no field named ownername, and only credentialowner
```	
// CredentialOwner is the owner of the model.
CredentialOwner user.Name
```
do you have any suggestion?

Reply:
Ah sorry I thought that we had this information in the model database. Looking at `GlobalName` we don't actually use it. The only place it is ever read is in a test.

Can I suggest that as part of this PR you just go through and remove this attribute off of the struct. There is 3 places by my count.

Reply:
fixed.